### DISCUSSION TITLE
Is it possible to get a value from a language via parameter of t()?

### DISCUSSION THREAD

**Discussion by kamerat** [sveltekit-i18n/lib#100] [Q&A] [answered]
I'm looking for a way to fetch some keys from different languages at the same place without changing the global locale.

For example:
```js
const greetings = {
  en: $t('common.hello', 'en'),
  no: $t('common.hello', 'no'),
  fr: $t('common.hello', 'fr'),
}
```

Or maybe another way to fetch all values from a key:

```js
const greetings = $t('common.hello').all()
```

Might be related to https://github.com/sveltekit-i18n/lib/issues/88

**Comment by jarda-svoboda** [comment, marked as answer]
Hi @kamerat.
Yes, it is. That's what is [`$l`](https://github.com/sveltekit-i18n/base/tree/b917e4fe27f7c31460c8f527b8a690a3ccf98c80/docs#l-readablelocale-string-key-string-vars-recordany-any--string---get-locale-string-key-string-vars-recordany-any--string-) born for..)
```svelte
{$l('locale', 'key', {/* props */})}
```
Don't forget to load all needed language mutations using `loadTranslations` method.

You can find an example [here](https://github.com/sveltekit-i18n/lib/tree/master/examples/single-load).

[a comment by an automated account is not shown]
